### ISSUE TITLE
mAP in voc2007

### ISSUE THREAD

**Issue by sorrowyn** [Yejin0111/ADD-GCN#5]
How to arrive the top mAP(96%)
but my best mAP is 92%. How can I overcome the problem?

args.seed = 1
args.lr = 0.05
args.image_size = 448
args.batch_size = 18 * gpu_num
args.epoch_step = [30, 40]

the test size is 576

**Comment by Yejin0111** [comment]
> How to arrive the top mAP(96%)
> but my best mAP is 92%. How can I overcome the problem?
> 
> args.seed = 1
> args.lr = 0.05
> args.image_size = 448
> args.batch_size = 18 * gpu_num
> args.epoch_step = [30, 40]
> 
> the test size is 576

I used the model that trained on COCO as the pre-train model for Pascal VOC. You can set ```--resume {Your model path that trained for COCO}``` for voc training.

**Comment by sorrowyn** [comment]
> 
> 
> > How to arrive the top mAP(96%)
> > but my best mAP is 92%. How can I overcome the problem?
> > args.seed = 1
> > args.lr = 0.05
> > args.image_size = 448
> > args.batch_size = 18 * gpu_num
> > args.epoch_step = [30, 40]
> > the test size is 576
> 
> I used the model that trained on COCO as the pre-train model for Pascal VOC. You can set `--resume {Your model path that trained for COCO}` for voc training.
Thanks senior. the accuracy of this experiment has been improved. Although it is still a little short of 96%, it is within the accepted range
